schemas/cloudToDevice/ground_fix/ground-fix.json

```diff
@@ -25,9 +25,29 @@
              },
              "fulfilledWith":{
                "$ref": "#/definitions/FulfilledWith"
+             },
+             "meta":{
```

Comment:
is the `"meta"` object necessary?  
can't `"anchors"` just go at the same level as the other data?

Reply:
The `meta` property helps separate the non location details into one handy property that can be easily ignored. It also allows us to put other details there in the future.

Reply:
It could possibly be flattened out. Does the extra property add too much overhead?

Reply:
i wouldn't say "too much", but yes it adds extra data and parsing.
from the device perspective it adds no value.

Reply:
What if we had a flag that could omit the metadata? There's no way for the device to use it now anyway. It could be true by default, so you would only get the meta prop if requested.

Reply:
i'm fine with the actual data being included, i'm just don't understand the _practical_ value of wrapping it in the "meta" object.

Reply:
It's for the API design. It logically separates the coordinates from the extra stuff that may come with it. Parsing this data becomes easier when all the extra stuff lives in one property, from an organizational perspective.

Reply:
sure, it is better "organized", but it just adds an extra step when parsing.
in this case, i don't see the anchor data as "extra" stuff.  it is very relevant to the location and how it was determined.
i would prefer that this be included without the "meta" wrapping.

Reply:
Okay. Refactored.